test/modules/errors/callModLikeProc2.good

```diff
@@ -1,2 +1,2 @@
 callModLikeProc2.chpl:7: In function 'main':
-callModLikeProc2.chpl:9: error: modules (like 'B' here) cannot be called like procedures
+callModLikeProc2.chpl:9: error: modules (like 'B' here) cannot be mentioned like variables
```

Comment:
I agree that this is an improvement, though it's not obvious that `A.B` is necessarily a variable reference.  E.g., if `B` was a procedure and this was intended to be an FCF expression, would 'variable' be the right term?

Reply:
I'm not sure if I'm understanding you correctly, but this error is specifically for module references in plain code (i.e., it only triggers when `B`, a module, is assigned to a variable or used in place of a variable or expression). Thus, if `B` was a procedure, this error would not trigger.

Reply:
My point was just that "like variables" feels either like a new term or like a restrictive view of what `A.B` could mean in Chapel.  I.e., if I wrote:

```chapel
module M {
  proc foo() { ... }
}
var myFCF = M.foo;
```

or:

```chapel
enum e { foo };
var s = e.foo;
```

am I "mentioning foo like a variable" (even though it isn't one)?  I guess on one hand it is "like how a variable might be mentioned", but it still feels like a limiting or potentially confusing description to me.  Now that I'm not racing for a plane, an alternative might be something like: "cannot serve as a standalone expression" or "is not a legal expression" or something like that...